<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appPreferencesDialog</class>
 <widget class="QDialog" name="appPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Application Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Recent Files</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="historySize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>40</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QCheckBox" name="showSplash">
     <property name="text">
      <string>Show splash screen when Companion starts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QCheckBox" name="startupCheck_companion9x">
     <property name="text">
      <string>Automatic check for Companion updates</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="QCheckBox" name="backupEnable">
     <property name="text">
      <string>Enable automatic backup before writing firmware</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Automatic Backup Folder</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="ge_lineedit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="ge_pathButton">
     <property name="text">
      <string>Find Executable</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QCheckBox" name="startupCheck_fw">
     <property name="text">
      <string>Automatic check for OpenTX firmware updates</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QCheckBox" name="wizardEnable_ChkB">
     <property name="text">
      <string>Offer to use wizard for new models</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ge_label">
     <property name="text">
      <string>Google Earth Executable</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>(Number of recent Models&amp;Settings files to remember)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QPushButton" name="backupPathButton">
     <property name="text">
      <string>Open Folder</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QLineEdit" name="backupPath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>historySize</zorder>
  <zorder>startupCheck_companion9x</zorder>
  <zorder>backupEnable</zorder>
  <zorder>label_17</zorder>
  <zorder>ge_lineedit</zorder>
  <zorder>line</zorder>
  <zorder>buttonBox</zorder>
  <zorder>backupPath</zorder>
  <zorder>line_2</zorder>
  <zorder>startupCheck_fw</zorder>
  <zorder>wizardEnable_ChkB</zorder>
  <zorder>ge_label</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>showSplash</zorder>
  <zorder>line_3</zorder>
  <zorder>ge_pathButton</zorder>
  <zorder>line_4</zorder>
  <zorder>backupPathButton</zorder>
 </widget>
 <resources>
  <include location="companion.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
